extensions-core/datasketches/src/main/java/io/druid/query/aggregation/datasketches/theta/SketchHolder.java

```diff
@@ -295,6 +296,14 @@ public boolean equals(Object o)
     if (o == null || getClass() != o.getClass()) {
       return false;
     }
-    return this.getSketch().equals(((SketchHolder) o).getSketch());
+    // Can't use Sketch's equals method because it is not implemented.
+    // return this.getSketch().equals(((SketchHolder) o).getSketch());
```

Comment:
Please don't include commented-out code - it should be deleted.

Reply:
Removed the commented-out codes.